src/PIL/ImageCms.py

```diff
@@ -787,14 +797,17 @@ def createProfile(colorSpace, colorTemp=-1):
         except (TypeError, ValueError) as e:
             msg = f'Color temperature must be numeric, "{colorTemp}" not valid'
             raise PyCMSError(msg) from e
+    else:
+        # colorTemp is unused if colorSpace != "LAB"
+        colorTemp = 0.0
```

Comment:
This is minor, but considering it's unused, what's the advantage of setting it to zero?

Reply:
The value is passed to `PyArg_ParseTuple` with the `d` format. Based on [the documentation](https://docs.python.org/3/c-api/arg.html#numbers), my understanding was that this required a `float` object, not a `SupportsFloat` object. However, testing with Python 3.11, it seems that a `SupportsFloat` object is actually accepted, so I'll revert this part of the change.

(Testing with `SupportsInt` I do get the expected `TypeError`, so at least that part matches the documentation.)